12FACTOR-APP.md

```diff
@@ -0,0 +1,30 @@
+ # The Twelve Factors:
```

Comment:
Formatting does not look good. Please change the formatting, remove ">>".
Please makes sure that the points are descriptive in the documentation.